packages/service-clients/azure-client/CHANGELOG.md

```diff
@@ -1,5 +1,60 @@
 # @fluidframework/azure-client
 
+## 2.12.0
```

Comment:
Same. I can see how this impacted the package internally, but the changelog entry feels weird here.